src/jobs/models/jobManager.js

```diff
@@ -8,7 +8,7 @@ const logger = require('../../common/logger'),
     databaseConnector = require('./database/databaseConnector');
 
 let cronJobs = {};
-const JOB_PLATFORM_NAME = 'predator.%s';
+const JOB_PLATFORM_NAME = 'predator-runner.%s';
```

Comment:
```suggestion
const JOB_NAME = 'predator-runner.%s';
```

this variable represents the job name

Reply:
i changed it to predator since there is limit of the name of the job to 63 chars
but i have found workaround using labels